tcbuilder/cli/build.py

```diff
@@ -348,6 +348,16 @@ def handle_raw_image_output(props, storage_dir, union_params, default_base_raw_i
 
     deploy_cli.deploy_raw_image(**deploy_raw_image_params)
 
+    handle_raw_image_bundle_output(
+        output_raw_img,
+        output_raw_img,
+        storage_dir,
+        props.get("bundle", {}),
+        props
+    )
```

Comment:
nit pick: how about indenting the same way as in `handle_easy_installer_output`.

Reply:
this one has two more arguments, I think that is better to read. But, this is a decision from the team, if you have a strict code style better to follow up.

Reply:
It's just a nit pick. Since it still fits into 100 columns we could follow the original indentation.

Reply:
You mean indenting like this?
```
    handle_raw_image_bundle_output(
        output_raw_img, output_raw_img, storage_dir, props.get("bundle", {}), props)
```

Reply:
Personally I'm fine with both styles, but I'll change the indentation as @rborn-tx suggested, just for the sake of consistency with the rest of the code.

Reply:
OK, done.